src/Menu.java

```diff
@@ -0,0 +1,137 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+public class Menu {
+
+
+    public static void startingMenu(){
+
+        try {
+            int userInput;
+            do {
+                Utils.clearScreen();
+                BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+                Utils.println("starting_menu", Game.isSoundOn()? "Off": "On");
+                userInput = Integer.parseInt(reader.readLine());
+                switch(userInput){
+                    case(2):
+                        //Learn about the trail
+                        learnAboutTrail();
+                        break;
+                    case(3):
+                        //See the top 10
+                        oregonTopTen();
+                        break;
+                    case(4):
+                        //Turn sound off
+                        Game.setSoundOn(!Game.isSoundOn());
+                        if(Game.isSoundOn()){
+                            System.out.println("Sound is now on");
+                        } else{
+                            System.out.println("Sound is now off");
+                        }
+                        break;
+                    case(5):
+                        //Choose Management Options
+                        managementMenu();
+                        break;
+                    case(6):
+                        // Exit the game
+                        System.exit(0);
```

Comment:
We should avoid System.out() calls. This will cause issues later on when the game had more ways of terminating and needs to clean stuff up at the end.

Reply:
Switched the main method from Game class to Menu class.  this allows the gameplay to be called from the menu, and when the user selects EXIT no system.exit() call needs to be made.